web/html/src/manager/audit/cveaudit/cveaudit.tsx

```diff
@@ -478,6 +478,24 @@ class CVEAudit extends React.Component<Props, State> {
                 }
               }}
             />
+            <Column
+              columnKey="scan-data"
+              width="15%"
+              comparator={Utils.sortByText}
+              header={t("Scan Data")}
+              cell={(row, criteria) => {
+                return (
+                  <div
+                    className="scan-data-indicator"
+                    style={{
+                      backgroundColor: row.scannedWithOVAL ? "#009688" : "orange",
```

Comment:
We have two different themes with their own branding colors so we can't use hardcoded color values like this. The easiest solution here would be to use classes such as `bg-success`, `bg-warning`, or `bg-danger`, would those suffice to do what you want to do here?